airflow-core/src/airflow/models/dagrun.py

```diff
@@ -190,6 +190,15 @@ class DagRun(Base, LoggingMixin):
     # Span context carrier, used for context propagation.
     context_carrier = Column(MutableDict.as_mutable(ExtendedJSON))
     span_status = Column(String(250), server_default=SpanStatus.NOT_STARTED, nullable=False)
+    created_dag_version_id = Column(
```

Comment:
Should this just be dag_version_id? It doesn't sound good having created_dag_version_id when there's no other fk to dag_version.

Reply:
the reason i chose to add "created" is because for a dag that is not i in a versioned bundle, there could be many dag versions associated with the dag run.  this makes it clear that this refers to the one that was in effect when the dag run was created.

Reply:
I think the many dag versions are handled at `dag_versions` method?  You can also see that the FK name is `dag_version_id_fkey`, should that also be named `created_dag_version_id_fkey`? Just for consistency with our db designs, I suggest we keep it as `dag_version_id`. Fwiw, TI.dag_version_id can also change lots of time with non-versioned bundle.

Reply:
The problem with calling it dag_version_id is, it is begs the question, "which dag version?".  It's ambiguous.  But, yes i missed updating the fk name thanks for catching that.